markdown/3.x/en/documentation/tracking-guide.md

```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
    <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
 </ff-checkout-tracking>
 ```
-If you do so, you can poke around with the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+If you want to send the tracking request to a channel different from the one configured in ff-communication, use the `channel` attribute.
```

Comment:
For consistency:
```suggestion
If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute.
```
For even more consistency, there are several mentions of `ff-` elements in regular text that could be unwrapped from `< >`. But I'll leave that to you to decide.